src/routers/router.tsx

```diff
@@ -12,28 +12,36 @@ import {
   DesignTestPage3,
   DesignTestPage4,
   DesignTestPage5,
-} from '@/pages/DesignTestPage'; /*
+} from '@/pages/DesignTestPage';
+import { ExperienceDetailPage } from '@/pages/ExperienceDetailPage';
 import { HomePage } from '@/pages/HomePage';
 import { LoginPage } from '@/pages/LoginPage';
-import { OnboardingPage } from '@/pages/OnboardingPage';*/
-import { RedirectPage } from '@/pages/RedirectPage'; /*
+import { OnboardingPage } from '@/pages/OnboardingPage';
+import { RedirectPage } from '@/pages/RedirectPage';
 import { SelfUnderstandPage } from '@/pages/SelfUnderstandPage';
-import { ExceptPreMemberRoute } from '@/routers/ExceptPreMemberRoute'; */
+import { ExceptPreMemberRoute } from '@/routers/ExceptPreMemberRoute';
+import { MemberPrivateRoute } from '@/routers/MemberPrivateRoute';
 
 export const Router = () => {
   return (
     <Routes>
-      {/* <Route element={<MainLayout />}>
-        <Route path="/onboarding" element={<OnboardingPage />} />
-        <Route element={<ExceptPreMemberRoute />}>
-          <Route path="/" element={<HomePage />} />
-          <Route path="/auth" element={<LoginPage />} />
-          <Route path="/understand" element={<SelfUnderstandPage />} />
-          // <Route element={<MemberPrivateRoute />}>
-            //<Route path="/mypage" element={<MyPage />} />
-          //</Route>
+      {
+        <Route element={<MainLayout />}>
+          <Route path="/onboarding" element={<OnboardingPage />} />
+          <Route element={<ExceptPreMemberRoute />}>
+            <Route path="/" element={<HomePage />} />
+            <Route path="/auth" element={<LoginPage />} />
+            <Route path="/understand" element={<SelfUnderstandPage />} />
+            <Route path="/recommend/detail/:id" element={<ExperienceDetailPage />} />
```

Comment:
지금 라우팅 설정으로 `/recommend/detail`만 입력하면 홈으로 팅겨나올 수 있어요. 그래서 이런 위험도 줄일 겸 `/detail`도 굳이 필요하지 않을 것 같아서 `recommend/:id`로 수정해도 좋을 것 같슴다.

```suggestion
            <Route path="/recommend/:id" element={<ExperienceDetailPage />} />
```